common/app/conf/switches/IdentitySwitches.scala

```diff
@@ -0,0 +1,69 @@
+package conf.switches
+
+import conf.switches.Expiry.never
+import org.joda.time.LocalDate
+
+trait IdentitySwitches {
+
+  val IdentityCookieRefreshSwitch = Switch(
+    SwitchGroup.Identity,
+    "id-cookie-refresh",
+    "If switched on, users cookies will be refreshed.",
+    owners = Seq(Owner.withGithub("johnduffell")),
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = true
+  )
+
+  val IdentityGdprMarketingConsentSwitch = Switch(
+    SwitchGroup.Identity,
+    "id-gdpr-marketing-consent",
+    "If switched on, users will post GDPR compliant marketing consent from 'Edit Profile' page",
+    owners = Seq(Owner.withGithub("mario-galic")),
+    safeState = Off,
+    sellByDate = new LocalDate(2018, 2, 1),
+    exposeClientSide = false
+  )
+
+  val IdentityAllowAccessToGdprJourneyPageSwitch = Switch(
+    SwitchGroup.Identity,
+    "id-allow-access-to-gdpr-journey-page",
+    "If switched on, users will be able to access the Journey page",
+    owners = Seq(Owner.withGithub("walaura")),
+    safeState = Off,
+    sellByDate = new LocalDate(2018, 2, 1),
+    exposeClientSide = false
+  )
+
+  val IdentityPointToConsentJourneyPage = Switch(
+    SwitchGroup.Identity,
+    "id-point-to-consent-journey-page",
+    "If switched on, public facing links will redirect to Journey page",
+    owners = Seq(Owner.withGithub("calum-campbell")),
+    safeState = Off,
+    sellByDate = new LocalDate(2018, 2, 1),
+    exposeClientSide = false
+
+  )
+
+  val IdentityRedirectUsersWithLingeringV1ConsentsSwitch = Switch(
```

Comment:
Arguably this could be merged into the switch above (point to consent journey).

Reply:
`point to consent journey ` will send emails tho, maybe a rename is in order?